src/main/java/org/kairosdb/core/datastore/KairosDatastore.java

```diff
@@ -57,6 +56,14 @@
     public static final String QUERY_SAMPLE_SIZE = "kairosdb.datastore.query_sample_size";
     public static final String QUERY_ROW_COUNT = "kairosdb.datastore.query_row_count";
 	public static final String METRIC_QUERY_CACHE_READ_TIME = "kairosdb.datastore.cache_read_time";
+	public static final String READ_TIMEOUT = "kairosdb.datastore.datapoints.read.timeout";
+
+	@Inject(optional = true)
+	@Named(READ_TIMEOUT)
+	private int m_readTimeout = 20000;
+
+	public int getReadTimeout(){return m_readTimeout;}
```

Comment:
Let's just kill the getter entirely - it's not on the interface, and not used by anything external to this class.